script/codesign/generate-identity.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+dir="$(dirname $0)"/.working
+
+cleanup() {
+    mkdir -p "$dir"
+    rm -r "$dir"
```

Comment:
no need for `mkdir` here, just `rm -rf`

Reply:
The `f` scared me a bit when running locally, I can change it for CI 😆